File: Python/MSSQL.py
from DB import DB

class MSSQL(DB):
    def selectData(self):
        print("Mssql veritabaninda veri cekme islemi gerceklestirilmistir.")

    def insertData(self):
        print("Mssql veritabaninda veri ekleme islemi gerceklestirilmistir.")

File: Java/MSSQL.java
public class MSSQL implements DB {
    @Override
    public void selectData() {
        System.out.println("Mssql veritabaninda veri cekme islemi gerceklestirilmistir.");
    }

    @Override
    public void insertData() {
        System.out.println("Mssql veritabainda veri ekleme islemi gerceklestirilmistir.");
    }
}